scripts/build_shapes.py

```diff
@@ -303,7 +303,47 @@ def eez(countries, country_shapes, EEZ_gpkg, out_logging=False, distance=0.01):
 
 
 def download_WorldPop(
-    country_code, year=2020, update=False, out_logging=False, size_min=300
+    worldpop_method,
+    country_code,
+    year=2020,
+    update=False,
+    out_logging=False,
+    size_min=300,
+):
+
```

Comment:
You could follow the docstring convention. Some empty spaces should be removed. See:
https://peps.python.org/pep-0257/